Dockerfile-prod

```diff
@@ -0,0 +1,27 @@
+FROM quay.io/sameersbn/ubuntu:latest
+MAINTAINER Alexander Trost <galexrt@googlemail.com>
+
+ENV ZULIP_VERSION="1.3.10" DATA_DIR="/data"
+
+RUN apt-get -q update && \
+    apt-get -q dist-upgrade -y && \
+    mkdir -p "$DATA_DIR" /root/zulip && \
+    wget -q "https://www.zulip.com/dist/releases/zulip-server-$ZULIP_VERSION.tar.gz" -O /tmp/zulip-server.tar.gz && \
+    tar xfz /tmp/zulip-server.tar.gz -C /root/zulip --strip-components=1 && \
+    rm -rf /tmp/zulip-server.tar.gz && \
+    export VOYAGER_CLASS="dockervoyager" DEPLOYMENT_TYPE="dockervoyager" \
+        ADDITIONAL_PACKAGES="python-dev python-six python-pbs" has_nginx="0" has_appserver="0" && \
+    /root/zulip/scripts/setup/install && \
+    chown zulip:zulip /opt/createZulipAdmin.sh && \
```

Comment:
It is a script that creates a zulip user with admin perms. The script is from my `galexrt/docker-zulip` repo.
Should I include it so Docker users have a simple way of creating an user?